index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Bassem shams portofolio</title>
+    <link rel="stylesheet" href="styles.css" type="text/css"/>
+</head>
+<body>
+  <div class="contain">
+      <header class="upper">
+        <img class="image"  src="assests/images/ic_menu.png" alt="the menu icon"></header>
+      <div class="content">
+          <h1 class="my-name">Bassem Shams</h1>
+          <p class="paragraph">
+              Hello! I am a software developer! I can help you build a product,
+             feature or website. Take a look of my works.
+             If you like what you see and have a project you need coded,
+             do not hesitate and contact me.</p>
+          <button class="button">Start collaboration</button>
+      </div>
```

Comment:
- [ ] Please make this section be in a `section` tag instead of the `div` tag you're using. It would be nice for all your sections to be in a `section` individually.